### PR TITLE
Added comment like and reply ui

### PR DESCRIPTION
**Pull request by RomanDeveloperAcc**
Added ui part related to the user story #776.
Link: https://github.com/ita-social-projects/GreenCity/issues/766

### CODE REVIEW

File: src/app/component/comments/comments.module.ts [hunk 1]
```diff
@@ -2,20 +2,20 @@ import { NgModule } from '@angular/core';
 import {CommentsComponent} from './comments/comments.component';
 import {SharedModule} from '../shared/shared.module';
 import {CommonModule} from '@angular/common';
-import {CommentsRoutingModule} from './comments-routing.module';
+import {CommentLikeReplyComponent} from './comments/comment-like-reply/comment-like-reply.component';
```
Comment (Gusvarov) [inline src/app/component/comments/comments.module.ts:5]: Please, add spaces between import brackets

File: src/app/component/eco-news/eco-news.module.ts [hunk 2]
```diff
@@ -26,6 +26,7 @@ import { ImageCropperModule } from 'ngx-image-cropper';
 import { EffectsModule } from '@ngrx/effects';
 import { EcoNewsEffects } from './store/eco-news.effects';
 import { EcoNewsSelectors } from './store/eco-news.selectors';
+import {CommentsModule} from '../comments/comments.module';
```
Comment (Gusvarov) [inline src/app/component/eco-news/eco-news.module.ts:30]: Same as previous comment

Comment (RomanDeveloperAcc) [reply src/app/component/eco-news/eco-news.module.ts:30]: Fixed

File: src/app/component/comments/comments.module.ts (hunk 1 above)
Comment (RomanDeveloperAcc) [reply src/app/component/comments/comments.module.ts]: Totally fixed

**Pull request merged**